docusaurus/docs/reactnative/common-content/contexts/messages-context/deleted_messages_visibility_type.mdx

```diff
@@ -0,0 +1,5 @@
+Controls the visibility of the deleted messages within the channel.
+
+| Type | Default |
+| - | - |
+| enum('both', 'none', 'receiver', 'sender') | 'both' |
```

Comment:
This should instead be imported, rather than duplicating the content between here and channel prop.